### PR TITLE
Feature/confirm words

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
This PR continues the onboarding process up through the educational carousel.

### CODE REVIEW

File: src/features/onboarding/accountCreatePin/AccountCreatePinScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+  const [pin, setPin] = useState('')
+  const [failedConfirmation, setFailedConfirmation] = useState(false)
+
+  useEffect(() => {
+    if (pin.length === 6) {
+      if (route.params?.pinReset) {
+        // TODO: Nav to ConfirmResetPin
+      } else {
+        navigation.push('AccountConfirmPinScreen', {
```
Comment (tyler-whitman) [inline src/features/onboarding/accountCreatePin/AccountCreatePinScreen.tsx:28]: Is there a place to store these route keys as variables so they are easier to look up and keep track of?

Not sure if the new react navigation has any guidelines around that?

File: src/features/onboarding/hotspotEducation/HotspotEducationScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    if (index === slides.length - 1) {
+      setViewedSlides(true)
+    }
+  }
+
+  const renderButton = () => {
+    if (viewedSlides) {
+      return (
+        <Button
+          // onPress={this.navNext}
```
Comment (tyler-whitman) [inline src/features/onboarding/hotspotEducation/HotspotEducationScreen.tsx:35]: Do we need a TODO here?

File: src/features/onboarding/hotspotEducation/HotspotEducationScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+          title={t('learn.next')}
+        />
+      )
+    }
+    return (
+      <Button
+        marginHorizontal="m"
+        variant="secondary"
+        mode="text"
+        // onPress={this.navNext}
```
Comment (tyler-whitman) [inline src/features/onboarding/hotspotEducation/HotspotEducationScreen.tsx:48]: Might need a TODO here as well

**Review by tyler-whitman — APPROVED** [review]
Nice! Looking good. Tested it out and it's working well.

**Comment by allenan** [comment]
noticed some weird sizing on the text in a couple of the carousel cards:

image

File: src/features/onboarding/OnboardingNavigator.tsx [hunk 4]
```diff
@@ -29,6 +33,22 @@ const Onboarding = () => {
...
       />
+      <OnboardingStack.Screen
+        name="AccountSecureScreen"
+        component={AccountSecureScreen}
+      />
+      <OnboardingStack.Screen
+        name="AccountCreatePinScreen"
+        component={AccountCreatePinScreen}
+      />
+      <OnboardingStack.Screen
```
Comment (allenan) [inline src/features/onboarding/OnboardingNavigator.tsx:44]: we might want to keep these screens out of the swipe back history

File: src/features/onboarding/accountCreatePin/AccountCreatePinScreen.tsx (hunk 1 above)
Comment (matthewcarlreetz) [reply src/features/onboarding/accountCreatePin/AccountCreatePinScreen.tsx:28]: For the onboarding flow there's a file `src/features/onboarding/onboardingTypes.ts` that has the types for this stack defined. You can type the `useNavigation` hook (eg. `const navigation = useNavigation<OnboardingNavigationProp>()`) and then you get type inference/checking around the routes and params. 

Here's a video that shows what I'm talking about...
https://share.getcloudapp.com/z8uYywl9

File: src/features/onboarding/hotspotEducation/HotspotEducationScreen.tsx (hunk 2 above)
Comment (matthewcarlreetz) [reply src/features/onboarding/hotspotEducation/HotspotEducationScreen.tsx:35]: yeah, good call. I'll add them.

File: src/features/onboarding/OnboardingNavigator.tsx (hunk 4 above)
Comment (matthewcarlreetz) [reply src/features/onboarding/OnboardingNavigator.tsx:44]: Do we want them to be able to go back at all after creating a pin? At that point they've finished their initial account setup - we could possibly push a modal for these last few steps or just clear the whole back stack?

Comment (allenan) [reply src/features/onboarding/OnboardingNavigator.tsx:44]: yeah I don't think they should be able to go back once they've finished this process

File: src/features/onboarding/accountCreatePin/AccountCreatePinScreen.tsx (hunk 1 above)
Comment (tyler-whitman) [reply src/features/onboarding/accountCreatePin/AccountCreatePinScreen.tsx:28]: Oh nice, no enum needed. Perfect

File: src/features/onboarding/OnboardingNavigator.tsx (hunk 4 above)
Comment (matthewcarlreetz) [reply src/features/onboarding/OnboardingNavigator.tsx:44]: ok - I'll work that into the next PR

**Review by allenan — APPROVED** [review]
looks good

**Pull request merged**